composer.json

```diff
@@ -45,6 +45,8 @@
         "jms/di-extra-bundle": "~1.7",
         "sensio/generator-bundle": "~2.3|~3.0",
         "symfony/yaml": "~2.3|~3.0",
+        "symfony/security-acl": "~2.4|~3.0",
+        "phpunit/phpunit": "~4.3",
```

Comment:
> Precision 8 is needed because tests are failiing otherwise

Why are we testing floats against something with precision 8 ? Like `assertEquals(0.5487, $sonataService->getMyFloat())` ?